### DISCUSSION TITLE
Help/assist in CSV import cities*

### DISCUSSION THREAD

**Discussion by TEST11R** [dr5hn/countries-states-cities-database#548] [Q&A]
I am using WP style sheet editor to import csv. Country.csv & States.csv worked together properly. But when iam importing cities it is just going randomly. Any idea how to do it properly. How did u guys did it? I need it for a taxonomy section.

**Comment by dr5hn** [comment]
@TEST11R No idea about importing CSV via WP Style sheet editor.

**Reply by TEST11R** [reply]
ok i have this as a taxonomy>>    listing> country> state

now i want to put city inside state..

How do we do it?
Whats your way?

Thanks

**Reply by TEST11R** [reply]
i just need this name & slug      parent>child>child-child
